.github/workflows/CI.yml

```diff
@@ -4,15 +4,13 @@ name: CI
 
 on:
   pull_request:
-    branches: [ master ]
+    branches: [ main, master ]
```

Comment:
using the name "main" and "master" together feels a bit confusing to me since both were used by github to indicate the protected branch. I prefer to use the branch name "dev" to increase clarity.